Automattic/wp-calypso
# Stats: Updating advanced feature check and CSS cleanup

Related to #87569

## Proposed Changes

* implementing feedback from https://github.com/Automattic/wp-calypso/pull/87813/
* aligning padding for stats modules without horizontal bars

## Testing Instructions



* open the live branch
* verify that `emails` and `subscribers` modules (Traffic and Subscribers pages) have items aligned with the module title (left padding has been removed)
* apply the UTM feature flag `stats/utm-module`
* verify that blogs with Jetpack and commercial plan can see the content of the UTM module


## Pre-merge Checklist
